tests/conftest.py

```diff
@@ -144,9 +144,10 @@ def ndq_series():
     time = xr.IndexVariable(
         "time", dates, attrs={"units": "days since 1900-01-01", "calendar": "standard"}
     )
+    rs = np.random.RandomState()
```

Comment:
Leaving `seed=None` will make a truly random `rs`,  while passing an int will make a fake random generator. Just saying in case you were trying to remove randomness from the tests.

Reply:
Good point.

On inspection, this fixture is never called. Perhaps we'll do a cull of the fixtures in another PR.